build.gradle

```diff
@@ -114,6 +114,10 @@ buildTypes {
         tasks "soak:soakTest"
         projectProperties testAllVersions: true
     }
+
+    forceRealizeDependencyManagementTest {
```

Comment:
I guess you don't want to add that change here, right?

Reply:
I saw there was a duplication between this and what we have on `master` so assumed it was required. It's not?